website/release_announcement_drafts/v1.3.0.md

```diff
@@ -0,0 +1,28 @@
+We're excited to announce the v1.3.0 release of JBrowse Web! Some highlights of
+this release include:
+
+## In-app plugin store
+
+Building on the plugin store on our website in the last release, we're now
+excited to announce that plugins can be installed from within JBrowse Web!
+Plugins from our plugin store can now be installed with the click of a button.
+
+![Screenshot of plugin store](https://user-images.githubusercontent.com/19295181/117894133-8dcc7480-b270-11eb-89fc-779be2e517a3.png)
```

Comment:
first screenshot may be unneeded IMO?

Reply:
i went ahead and removed...can revert if needed but seemed nice and minimal and people should still get to see the sidebar